src/editor/editor.rs

```diff
@@ -173,6 +175,12 @@ impl Editor {
                 self.status_message =
                     StatusMessage::from("Save as: ");
             },
+            Key::Ctrl('d') => {
```

Comment:
this should probably only be available in debug mode. it's just useful for debugging the compiler from the in-game terminal